### PR TITLE
AP-3965 Vault fetch

### PR DESCRIPTION
**Pull request by kamilwylegala**
## Changes

Script utility for fetching secrets from provided vault instance + merging them into configured .env file.

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/app/script-utils/vite.config.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { resolve } from 'path'
```
Comment (kibertoad) [inline packages/app/script-utils/vite.config.ts:1]:
```suggestion
import { resolve } from 'node:path'
```

Comment (kamilwylegala) [reply packages/app/script-utils/vite.config.ts]: Done. It was copied from other packages. Let me change it everywhere.

Comment (kamilwylegala) [reply packages/app/script-utils/vite.config.ts]: Changed in other places too :blush:

**Review by kibertoad — APPROVED** [review]
Thank you, this is awesome!

**Comment by kamilwylegala** [comment]
> @kamilwylegala could you please add the release workflow as we have for the other libs 🙏? You can take the others as examples and let me know if you have problems

Sure. let me check what it is about.

**Comment by CarlosGamero** [comment]
Outstanding! Thank youuu 🙌

File: packages/app/script-utils/src/vault/syncEnvWithVault.spec.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import { afterEach, describe, expect, it } from 'vitest'
+import { updateEnvFile } from './syncEnvWithVault'
+import { unlinkSync, readFileSync, writeFileSync, existsSync } from 'node:fs'
+
+const DOT_ENV_PATH = __dirname + '/test_env'
+
+function readDotEnvFile() {
+	let rawcontent = readFileSync(DOT_ENV_PATH, { encoding: 'utf8' })
+
+	console.log(rawcontent)
```
Comment (jmontoyaa) [inline packages/app/script-utils/src/vault/syncEnvWithVault.spec.ts:10]: Is this expected? :thinking:

**Review by jmontoyaa — APPROVED** [review]

**Pull request merged**